File: experiments/llm-reasoning-cost/generated_code/java/03_vector_ops_codex.java
import java.util.ArrayList;
import java.util.List;

public class Main {
    public static int sum_positive(List<Integer> nums) {
        List<Integer> positives = new ArrayList<>();

        for (int i = 0; i < nums.size(); i++) {
            int value = nums.get(i);
            if (value > 0) {
                positives.add(value);
            }
        }

        int sum = 0;
        for (int i = 0; i < positives.size(); i++) {
            sum += positives.get(i);
        }

        return sum;
    }
}

File: experiments/llm-reasoning-cost/generated_code/go/03_vector_ops_codex.go
package main

func sum_positive(nums []int32) int32 {
	positives := make([]int32, 0)

	for _, n := range nums {
		if n > 0 {
			positives = append(positives, n)
		}
	}

	var sum int32 = 0
	for _, n := range positives {
		sum += n
	}

	return sum
}